presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -45,6 +46,11 @@ public HiveSessionProperties(HiveClientConfig config)
     {
         sessionProperties = ImmutableList.of(
                 booleanSessionProperty(
+                        BUCKET_EXECUTION_ENABLED,
+                        "Assign one bucket to each worker",
```

Comment:
In this case the reason to add it to session variable is because when there are very few buckets execution will slow down rather than speed up when enabled. So the config description is not really accurate.